src/OrchardCore.Modules/OrchardCore.Media.AmazonS3/Startup.cs

```diff
@@ -18,116 +18,117 @@
 using OrchardCore.Media.Events;
 using OrchardCore.Modules;
 
-namespace OrchardCore.Media.AmazonS3;
-
-public class Startup : Modules.StartupBase
+namespace OrchardCore.Media.AmazonS3
```

Comment:
Why put back the brackets for the namespace?

Reply:
Just for consistency with other code, otherwise I will revert this change and I might work on a new PR to use the new feature

Reply:
I see. Good point, you can leave this then (I see there was no actual change apart from this) and please open an issue for moving the whole app to file scoped namespaces.